### PR TITLE
Add coq-topology.10.0.1 and coq-zorns-lemma.10.0.1

### PR DESCRIPTION
**Pull request by Columbus240**

### CODE REVIEW

File: released/packages/coq-topology/coq-topology.10.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+maintainer: "palmskog@gmail.com"
+version: "dev"
```
Comment (gares) [inline released/packages/coq-topology/coq-topology.10.0.1/opam:3]: this is incorrect

File: released/packages/coq-zorns-lemma/coq-zorns-lemma.10.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+maintainer: "miloradovsky@gmail.com"
+version: "dev"
```
Comment (gares) [inline released/packages/coq-zorns-lemma/coq-zorns-lemma.10.0.1/opam:3]: This is wrong, the version is inferred from the directory name

File: released/packages/coq-zorns-lemma/coq-zorns-lemma.10.0.1/opam [hunk 3]
```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+maintainer: "miloradovsky@gmail.com"
```
Comment (gares) [inline released/packages/coq-zorns-lemma/coq-zorns-lemma.10.0.1/opam:2]: Is this you?

**Review by gares — CHANGES_REQUESTED** [review]

File: released/packages/coq-topology/coq-topology.10.0.1/opam (hunk 1 above)
Comment (palmskog) [inline released/packages/coq-topology/coq-topology.10.0.1/opam:3]:
```suggestion
```

File: released/packages/coq-zorns-lemma/coq-zorns-lemma.10.0.1/opam (hunk 3 above)
Comment (palmskog) [inline released/packages/coq-zorns-lemma/coq-zorns-lemma.10.0.1/opam:2]:
```suggestion
maintainer: "palmskog@gmail.com"
```

File: released/packages/coq-zorns-lemma/coq-zorns-lemma.10.0.1/opam (hunk 2 above)
Comment (palmskog) [inline released/packages/coq-zorns-lemma/coq-zorns-lemma.10.0.1/opam:3]:
```suggestion
```

**Pull request merged**